src/malloc/lite_malloc.c

```diff
@@ -103,7 +103,9 @@ static void *__simple_malloc(size_t n)
 	return p;
 }
 
-weak_alias(__simple_malloc, __libc_malloc_impl);
+// PH: temporary eliminate use of weak-alias until the repo tools fully support
+// symbol aliasing.
+// weak_alias(__simple_malloc, __libc_malloc_impl);
```

Comment:
What will happen if a function calls `__libc_malloc_impl`?

Reply:
It will call the correct code (written as malloc, but this is #defined to __libc_malloc_impl, in malloc/mallocng/malloc.c) rather than the wrong code (__simple_malloc in lite_malloc.c). That’s the fix!

Reply:
Thanks Paul.